config.md

```diff
@@ -265,6 +265,19 @@ Annotations are key-value maps.
 }
 ```
 
+## ImagePath
+
+ImagePath is an optional absolute path to the root filesystem image used by VM-based runtimes.
+
+## Kernel-configuration
+
+Used by VM-based runtimes only.
+
+* **`path`** (string, required) specifies an absolute path to the kernel
+  used to boot the virtual machine.
+* **`parameters`** (string, optional) specifies a space-separated list of parameters to pass to the kernel.
+* **`initrd`** (string, optional) specifies an absolute path to an initial ramdisk to be used by the kernel.
```

Comment:
Looking through the existing docs, the only paths I can see that represents entities that must exist but which are allowed to be relative are:

- `process.args[0]`: this is reasonable given `$PATH` can be searched plus it permits the command to be absolute.

- `root.path`: this is potentially reasonable given that that path must exist below `$bundle_dir`.
  However, [the runtime start command](https://github.com/opencontainers/runtime-spec/blob/master/runtime.md#start) does not specify that the `start` command be passed an absolute path as it's second argument (`start <container-id> <path-to-bundle>`). This seems surprising - if it's not absolute, how is the runtime to find it? Is this an oversight or is there a contract somewhere stating that the runtime will always be launched from, or set its cwd to "`$bundle_dir`"?

There are only two points of reference that a runtime has from what I can see:

- it's cwd.
- the bundle directory provided to it.

The elements referenced by this PR (`kernelConfig.path`, `kernelConfig.initrd` and `imagePath`):

- are not commands so cannot be found using `$PATH`.
- will not be available below `$bundle_dir`.

Hence, I believe they must be absolute paths. Further, it's safer to specify absolute paths. On that topic, I note that all hook paths must be absolute. Also, absolute paths are more in line with the imperative nature of the configuration file - don't give the runtime choices (unless there's are good reason to ;-)

Since paths can be troublesome, I wonder if all paths mentioned by the spec should be reviewed to ensure that for each path that is permitted to be relative, a rationale be provided as to why it can be relative?